### PR TITLE
Bump up version of eslint-config-standard

### PR DESCRIPTION
**Pull request by marusak**
Current version was a year old. It updated a few things, mostly:

* No spaces in square brackets
* Use `const` when value never changes
* Don't use `hasOwnProperty`

Mostly fixed with `npm run eslint:fix` with some manual changes required.

And if/when this lands we are gonna have some conflicts for sure with existing PRs, but I guess we need to count with that

### CODE REVIEW

**Comment by marusak** [comment]
When I update everything `eslint` related it introduces 16 new (or stronger) rules. Few of them are useful, which would make sense for us. Other maybe could be just disabled (at least for now).

**Comment by marusak** [comment]
> Why is that "needswork"

@Gundersanne was not that happy that we introduce a lot of whitespace fixes. Also if I would update `eslint-config-standard-react` and other `eslint*` packages, it would end up with a lot more fixes like these.

Question is whether we want to do:
1. Update all eslint packages at once and:
1.1 Fix everything
1.2 Disable most of the new rules (like no-spaces-in-[]) and fix only those, that could do some harm or do not bring that many changes
2. Keep only this update and:
2.1 Keep as is
2.2 Split to more commits by type of fix (every rule a separate commit).

**Comment by martinpitt** [comment]
My approach with the last round was:
 * Land everything in one big and quick PR that can be auto-fixed (eslint:fix), i. e. mostly whitespace and other trivial changes.
 * Enable the other rules one by one in separate PRs/commits and fix the associated issues.

So personally I'm fine with landing this now, this shouldn't get dragged out too much. Of course it will most likely cause other PRs to conflict, but we never have a time with very few PRs open, so it pretty well doesn't matter.

**Comment by marusak** [comment]
> Land everything in one big and quick PR that can be auto-fixed

makes sense. I'll rework this a bit - I'll add all eslint new packages, fix those basic stuff and disable others - then we can fix those one by one.

**Comment by marusak** [comment]
So I pushed here all changes that were possible to fix with `eslint:fix` or with little effort. Every change is separate commit.
`+5,572 −4,998` changes now, sorry @Gundersanne :D 
But majority of those changes are by 2 or three commits. And I am fine with dropping those and keeping those permanently disabled. What do you think @martinpitt 

Also from those 5 rules that are disabled, first three (`react/jsx-key`, `react/jsx-handler-names` and `react/jsx-no-target-blank`) should be definitely fixed. But as follow up.

**Comment by marusak** [comment]
And yes, this is gonna get conflicts vary fast and very likely gonna case conflicts with other PRs. Also it is possible that we land something which later shows up breaks this - we just need to count on that, there is not much we can do about it.

**Comment by marusak** [comment]
So 4 commits make 90% of all changes:
`Object curly brackets should be alone on a line` is almost 40% of all changes
`There should be no space after [ and before ]` is almost 20% of all changes
`Don't quote properties names` is around 15% of all changes
`Prefer "const" to "let"` is around 15% of all changes

File: pkg/apps/application-list.jsx [hunk 1]
```diff
@@ -68,7 +68,8 @@ class ApplicationRow extends React.Component {
                     <div>
                         {comp.summary}
                         <div className="alert alert-danger alert-dismissable">
-                            <button className="close"
+                            <button
+className="close"
```
Comment (martinpitt) [inline pkg/apps/application-list.jsx]: This looks really broken. Also, this is a rule which is much more ambiguous than the others  so far -- it's not really harder to read in the old way. Can we revert this one for now and disable the rule, and revisit this in a follow-up?

File: pkg/lib/cockpit-components-logs-panel.jsx [hunk 2]
```diff
@@ -50,12 +50,10 @@ class JournalOutput {
                 <div className="cockpit-log-warning" role="cell">
                     { warning
                         ? <i className="fa fa-exclamation-triangle" />
-                        : null
-                    }
+                        : null}
```
Comment (martinpitt) [inline pkg/lib/cockpit-components-logs-panel.jsx]: This is another debatable one -- now it's harder to see the corresponding opening bracket. Maybe let's revert/disable that one as well for now?

**Comment by marusak** [comment]
Dropped those two commits and rebased to current master.

**Comment by martinpitt** [comment]
[selenium-chrome](https://209.132.184.41:8493/logs/pull-12644-20190830-062424-f86a8389-cockpit-project-cockpit--fedora-30-selenium-chrome/log.html) is almost certainly a flake, but retrying.

**Pull request merged**